### PR TITLE
Update to transaction_ignore_urls config name

### LINKED ISSUE #1313: Add TRANSACTION_IGNORE_URLS option
See https://github.com/elastic/apm/issues/144

### PR DESCRIPTION
**Pull request by eyalkoren**
Closes #1313 

@felixbarny I believe all we are required to do is renaming, while still allowing the old name so it is not a breaking change, right?
cc @gregkalapos 🙂

### CODE REVIEW

File: apm-agent-core/src/main/java/co/elastic/apm/agent/impl/context/web/WebConfiguration.java [hunk 1]
```diff
@@ -58,7 +58,8 @@
 
     private final ConfigurationOption<List<WildcardMatcher>> ignoreUrls = ConfigurationOption
         .builder(new ListValueConverter<>(new WildcardMatcherValueConverter()), List.class)
-        .key("ignore_urls")
+        .key("transaction_ignore_urls")
+        .aliasKeys("ignore_urls")
```
Comment (gregkalapos) [inline apm-agent-core/src/main/java/co/elastic/apm/agent/impl/context/web/WebConfiguration.java:62]: Wow, you can even do an alias in your config-system; very fancy! 👏 👏

Comment (eyalkoren) [reply apm-agent-core/src/main/java/co/elastic/apm/agent/impl/context/web/WebConfiguration.java:62]: Yeah, we use a fancy third party for that 😉

Comment (felixbarny) [reply apm-agent-core/src/main/java/co/elastic/apm/agent/impl/context/web/WebConfiguration.java:62]: Let's call it a 2.5 party 😛

**Comment by felixbarny** [comment]
@eyalkoren could you create a changelog for this?

**Comment by eyalkoren** [comment]
> could you create a changelog for this?

@felixbarny it was added before merging. You probably looked at the RC release notes and not the [unreleased](https://www.elastic.co/guide/en/apm/agent/java/master/_unreleased.html#_features)

**Comment by felixbarny** [comment]
ah, right 🤦‍♂️

**Pull request merged**